variants/arduino_zero/linker_scripts/gcc/flash_with_bootloader.ld

```diff
@@ -67,6 +67,9 @@ SECTIONS
 	{
 		__text_start__ = .;
 
+		KEEP(*(.ota_boot))
+
+		. = ALIGN(256);
```

Comment:
I'd align the .ota_boot section to either 0x2000 or 0x4000 to make sure that the reset vector does not get relocated to something different than 0x6000 if the 2nd stage bootloader becomes slightly smaller